frontend/package.json

```diff
@@ -23,6 +24,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
+    "server": "nodemon index.js --ignore client",
```

Comment:
Got an error about not being able to use a proxy. Stack Overflow suggested this.